taxi_service/settings.py

```diff
@@ -50,6 +51,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
The `DebugToolbarMiddleware` should only be included in `MIDDLEWARE` when `DEBUG` is `True`. Consider making this conditional to avoid exposing sensitive information in production.